test/powershell/Modules/Microsoft.PowerShell.Utility/Invoke-Item.Tests.ps1

```diff
@@ -67,7 +67,8 @@ Describe "Invoke-Item basic tests" -Tags "Feature" {
                 Get-Process -Name $notepadProcessName | Stop-Process -Force
                 Invoke-Item -Path $notepad
                 $notepadProcess = Get-Process -Name $notepadProcessName
-                $notepadProcess.Name | Should Be $notepadProcessName
+                # we need BeIn because multiple notepad processes could be running
+                $notepadProcess.Name | Should BeIn $notepadProcessName
```

Comment:
In line 67 we kill all notepad processes.

Reply:
in CI this is probably ok - on a dev box, not so much - the fix for this should be part of the generalized test review